src/Sylius/Bundle/AttributeBundle/spec/Form/Type/AttributeType/Configuration/SelectAttributeConfigurationTypeSpec.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\AttributeBundle\Form\Type\Configuration\AttributeType;
+
+use PhpSpec\ObjectBehavior;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class SelectAttributeConfigurationTypeSpec extends ObjectBehavior
+{
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\AttributeBundle\Form\Type\AttributeType\Configuration\SelectAttributeConfigurationType');
+    }
+
+    function it_is_abstract_type()
+    {
+        $this->shouldHaveType(AbstractType::class);
+    }
+
+    function it_builds_configuration_form(FormBuilderInterface $builder)
+    {
+        $builder
+            ->add('options', 'collection', [
+                'type' => 'text',
+                'label' => 'sylius.form.attribute_type_configuration.select.values',
+                'allow_add' => true,
+                'allow_delete' => true,
+            ])
+            ->add('multiple', 'checkbox', array('label' => 'sylius.attribute_type_configuration.select.multiple'))
+            ->willReturn($builder)
```

Comment:
However I think it's a discussable topic (I think that they're useful, as they ensure form type configuration) and it should be done for now ;)